Homepage

Liliya-V opened the pull request.

Liliya-V committed:
Homepage try1

Liliya-V committed:
commiting all files which have been used in a template build

Liliya-V committed:
changed name of homepage.html to template.html

Liliya-V committed:
1.added     nav {
        display: flex;
        flex-direction: column;
so on small device navbar goes vertical
2. set a social icon size relative to a veiwport

mporam commented on .idea/vcs.xml:
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
this file needs to be in your gitignore

mporam commented on homepage.html:
```diff
@@ -0,0 +1,40 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
```
remove the .html from the title

mporam commented on homepage.html:
```diff
@@ -0,0 +1,40 @@
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio Liliya Voevodina.html</title>
+        <link href="normalizeportfolio.css" rel="stylesheet" type="text/css">
```
can you name this file normalize.css - to keep it consistant

mporam commented on homepage.html:
```diff
@@ -0,0 +1,40 @@
+        <title>Portfolio Liliya Voevodina.html</title>
+        <link href="normalizeportfolio.css" rel="stylesheet" type="text/css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link href="stylesheet.css" rel="stylesheet" type="text/css">
```
can you put all your css files into a css folder please?

mporam commented on homepage.html:
```diff
@@ -0,0 +1,40 @@
+    <body>
+        <div class="hometop">
+            <nav>
+                <img class="mypicture" src="IMG_9169.jpg" alt="Liliya">
```
can you put your images into an images folder

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+*{
```
`* {` not `*{`

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    height: 100vh;
+    width: 100vw;
+    position: relative;
+
```
remove blank line

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+}
+
+body {
+    background-color :#f2f2f2;
```
`background-color: #f2f2f2;` not `background-color :#f2f2f2;`

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    width: 100vw;
+    position: relative;
+
+}
```
missing line break between `}` and new selector

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    padding: 10px 10px 0 10px;
+    margin: 5px 5px 0 5px;
+    position:relative;
+
```
remove line

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+
+}
+nav {
+    width: 98%;
```
why 98% and not 100%?

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    position:relative;
+
+}
+nav {
```
missing line break before selector

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+.mypicture {
+    width: 100px;
+    height: 100px;
+    border: 2px solid cornflowerblue;
```
please use a hex instead of the colour name

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    margin: 14px;
+}
+a:hover {
+    color: cornflowerblue;
```
use a hex

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    /*margin: 0 auto;*/
+    width: 100%;
+    height: 90%;
+    /*display: flex;*/
```
remove any commented code

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    height: 90%;
+    position: absolute;
+    left: 20%;
+
```
remove line

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+    bottom: 15px;
+    left: 25%;
+    margin: 0;
+
```
remove line

mporam commented on stylesheet.css:
```diff
@@ -0,0 +1,106 @@
+.social {
+    display:flex;
+    flex-direction: column;
+
```
remove

mporam reviewed (changes requested):
please make sure your css follows the following format always:

```
selector {
    property: value;
}

selector {
    property: value;
}

Liliya-V committed:
Adjested CSS to a correct format,
Removed extra lines,
Removed .HTML from the title,
renamed normalizeportfolio.css to normalize.css.

Liliya-V committed:
removed extra line

mporam commented on css/stylesheet.css:
```diff
@@ -0,0 +1,108 @@
+
+nav a {
+    text-decoration: none;
+    font-size: 2vh;
```
why is this a vh value? that means it will shrink with the screen, is that what you want?

Liliya-V replied:
Done it on purpose so my text is responsive

mporam reviewed (changes requested):
you need to update your html with the new CSS and image locations, as they dont currently work

Liliya-V committed:
moving images and css into folders

Liliya-V committed:
creating gitignot and removing a .idea folders

Liliya-V committed:
renamed normalize css in my HTML

Liliya-V committed:
changing css files and images links so it sees them into their directories

Liliya-V committed:
linked about me button to aboutme.html
